ui/src/features/dag-runs/components/dag-run-details/DAGRunDetailsModal.tsx

```
@@ -38,6 +38,11 @@ function DAGRunDetailsModal({
   const [shouldRender, setShouldRender] = useState(isOpen);
   const [isVisible, setIsVisible] = useState(false);
   const previousDataRef = useRef<PreviousData | null>(null);
+  const prevRemoteNodeRef = useRef(remoteNode);
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)